config/environments/production.rb

```diff
@@ -34,24 +33,34 @@
   # config.asset_host = "http://assets.example.com"
 
   # Specifies the header that your server uses for sending files.
-  config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
+  # config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
```

Comment:
I think we want to set this header: https://github.com/sul-dlss/stacks/pull/898
```suggestion
  config.action_dispatch.x_sendfile_header = "X-Sendfile" # for Apache
```

Reply:
Thanks! Got it fixed.